duckduckgo/iOS
# improve error UI in landscape

Task/Issue URL: https://app.asana.com/0/414235014887631/615601668272721
Tech Design URL:
CC:

**Description**:

Fixes UI when in landscape mode for the error page and the error view on the dashboard.

Uses a similar to technique to vertical padding PR by resizing the table footer dynamically.  It's not perfect, because there is white space when the retry button is hidden, but it's better.  Trying to hide that cell causes the footer to overlap with the rest of the table when in landscape. 😱 

**Steps to test this PR**:

Test 1
1. Fresh install, load the app with wifi off and get through on boarding.
1. Turn wifi on and visit a page
1. Open privacy dashboard to see error and try again button. Check this in portrait and landscape.

Test 2
1. Browse to a page that doesn't exist
1. Observer error messages, checking portrait and and landscape.
1. Open dashboard and check error message there in portrait and landscape.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
